src/common/hooks/use-network-mode.ts

```diff
@@ -2,6 +2,6 @@ import { useRecoilValue } from 'recoil';
 import { networkModeState } from '@pages/_app';
 
 export const useNetworkMode = () => {
-  const state = useRecoilValue(networkModeState);
+  const state = useRecoilValue<'Testnet' | 'Mainnet' | null>(networkModeState);
```

Comment:
Do we want to have the capitalised value as the key in the store, versus lowercase key and formatting later to capitalise?

Reply:
fixed, thank you